contracts/connectors/loantoken/LoanToken.sol

```diff
@@ -29,7 +29,7 @@ contract LoanToken is AdvancedTokenStorage {
             return;
         }
 
-        address target = target_;
+        address target = _getTarget();
```

Comment:
how is this getTarget funciton impoving  things?

Reply:
LoanTokenFactory.sol overrides it to pull the target from the factory contract itself. This allows for setting the target in a singular place and affecting all the factory tokens. Factory token logic differs slightly from non-factory as factory tokens will all use the same demand curves and flash loan fee.